htdocs/admin/dict.php

```diff
@@ -1261,6 +1262,10 @@
 
 llxHeader('', $title, '', '', 0, 0, '', '', '', 'mod-admin page-dict');
 
+$head = socialnetwork_prepare_head();
+
+print dol_get_fiche_head($head, 'dict', $langs->trans('MenuDict'), -1, 'user');
```

Comment:
If you add a dol_get_fiche_head, you must also add a dol_get_fiche_end().

Also adding the tab must be introduce only when you open the page from the page of the module social network.
When this page is open from menu Home - dictionnary, we must keep only the simple table view with no tabs.

Reply:
ok i see i will fix it